lib/dialects/mysql/query.js

```diff
@@ -110,6 +110,8 @@ Query.prototype.formatResults = function(data) {
     result = data[0].version;
   } else if (this.isForeignKeysQuery()) {
     result = data;
+  } else if (this.isInsertQuery() || this.isUpdateQuery()) {
+    result = [result, data.affectedRows];
```

Comment:
For `MySQL` you might want to use `changedRows` which is the actual number of rows updated, rather than `affectedRows` which is number of rows selected by query, which make difference when working on `update` statements